src/pages/documentation/current-sdk-versions.md

```diff
@@ -6,6 +6,10 @@
 
 **Migrate to latest Mobile SDKs for Android**<br/>If you are currently using the Mobile Core 1.x and the compatible libraries, please see the [Migrating to latest Mobile SDKs for Android](https://developer.adobe.com/client-sdks/previous-versions/documentation/migrate-to-android/) guide for next steps.<br/>The latest Mobile SDKs for Android support Google Android API 19 (KitKat) or later.
 
+<InlineAlert variant="info" slots="text"/>
+
+**Migrate to using the BOM artifact for Android**<br/> The Adobe SDK BOM artifact enables managing all compatible versions of Adobe Experience Platform Android extensions by specifying a single BOM version. This is now the recommended way to manage Android SDKs.
```

Comment:
Migration is overloaded here, can we use:
Migrate to using the BOM ... -> Start using the BOM ...